system/lib/libc/musl/include/unistd.h

```diff
@@ -198,6 +198,7 @@ char *get_current_dir_name(void);
 int syncfs(int);
 int euidaccess(const char *, int);
 int eaccess(const char *, int);
+pid_t gettid(void);
```

Comment:
should this be marked as a local mod?

Reply:
I mean its backported from musl tot..  maybe not worth marking?

Reply:
Oh, i got it. yeah if it's upstream on trunk it seems fine.